diff -up chrono-0.4.19/Cargo.toml.1~ chrono-0.4.19/Cargo.toml
--- chrono-0.4.19/Cargo.toml.1~	2020-09-30 17:06:24.000000000 +0200
+++ chrono-0.4.19/Cargo.toml	2021-03-27 23:10:24.722885085 +0100
@@ -72,9 +72,6 @@ optional = true
 [dev-dependencies.bincode]
 version = "0.8.0"
 
-[dev-dependencies.criterion]
-version = "0.3"
-
 [dev-dependencies.doc-comment]
 version = "0.3"
 
@@ -93,25 +90,11 @@ version = "1"
 __doctest = []
 __internal_bench = []
 alloc = []
-clock = ["libc", "std", "winapi"]
+clock = ["libc", "std"]
 default = ["clock", "std", "oldtime"]
 oldtime = ["time"]
 std = []
 unstable-locales = ["pure-rust-locales", "alloc"]
-wasmbind = ["wasm-bindgen", "js-sys"]
-[target."cfg(all(target_arch = \"wasm32\", not(any(target_os = \"emscripten\", target_os = \"wasi\"))))".dependencies.js-sys]
-version = "0.3"
-optional = true
-
-[target."cfg(all(target_arch = \"wasm32\", not(any(target_os = \"emscripten\", target_os = \"wasi\"))))".dependencies.wasm-bindgen]
-version = "0.2"
-optional = true
-[target."cfg(all(target_arch = \"wasm32\", not(any(target_os = \"emscripten\", target_os = \"wasi\"))))".dev-dependencies.wasm-bindgen-test]
-version = "0.3"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.0"
-features = ["std", "minwinbase", "minwindef", "timezoneapi"]
-optional = true
 [badges.appveyor]
 repository = "chronotope/chrono"
 
